### PR TITLE
chore(makefile): Update makefile to push canary tag when building new image

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

**Comment by vdice** [comment]
As mentioned in the 'sibling' PR (https://github.com/deis/logger/pull/47), I might vote we instead bring in a `versioning.mk` like the other deis components use (for example, https://github.com/deis/controller/blob/master/versioning.mk)

File: versioning.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+.PHONY: docker-push
+docker-push: docker-mutable-push docker-immutable-push
+
+.PHONY: docker-immutable-push
+docker-immutable-push:
+	docker push ${IMAGE}
+
+.PHONY: docker-mutable-push
+docker-mutable-push:
+	docker push ${MUTABLE_IMAGE}
```
Comment (arschles) [inline versioning.mk:22]: nit: can you add a newline here?

**Comment by arschles** [comment]
code LGTM, except for [my nitpick](https://github.com/deis/fluentd/pull/13#discussion_r58928773). I agree with @vdice's [comment](https://github.com/deis/fluentd/pull/13#issuecomment-207020073) about bringing in the standard `versioning.mk` file, however. Maybe file an issue for that?

**Comment by jchauncey** [comment]
already did that.

**Pull request merged**